### DISCUSSION TITLE
How to deal with minimization of objectives?

### DISCUSSION THREAD

**Discussion by RossXli** [meta-pytorch/botorch#1258] [Q&A] [answered]
In my case to minimize one of objectives, do I need to multiply the objective training data by -1 in advance for model initialization since most AF only deal with maximization problems.

**Comment by eytan** [comment, marked as answer]
Yes.

**Reply by saitcakmak** [reply]
Alternatively, you can pass a `WeightedMCMultiOutputObjective` to the acquisition function with weights denoting the optimization direction for each objective (1 for maximization, -1 for minimization`: https://github.com/pytorch/botorch/blob/main/botorch/acquisition/multi_objective/objective.py#L110
